### DISCUSSION TITLE
Updated to the new version with "docker compose pull"

### DISCUSSION THREAD

**Discussion by vmsman** [netalertx/NetAlertX#790] [Q&A]
Updated to the new version with "docker compose pull".  When I restarted the app "docker compose up -d" and reloaded the cache as indicated, my 160 records were gone and the app lost 100% of my configuration.

**Comment by jokob-sk** [comment]
Hi @vmsman, I hope I can help.

Can you please check if there is a backup available as per https://github.com/jokob-sk/NetAlertX/blob/main/docs/BACKUPS.md? The devices.csv file would help you to get all your devices data back.

Also, can you please double check the maintenance section of the DB stats, I'm wondering if it's a UI issue or if the DB really is empty.

Hope I can help you get your data back,
J
